### PR TITLE
add multi modif in carousel + modif majeur in css

### PR DESCRIPTION
**Pull request by NtGZeko**

### CODE REVIEW

**Review by Dash293 — DISMISSED** [review]

File: backend/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
     "cors": "^2.8.5",
     "dotenv": "^10.0.0",
     "express": "^4.17.2",
-    "mysql2": "^2.3.3"
+    "mysql2": "^2.3.3",
+    "template-fullstack": "file:.."
```
Comment (LoicBrassart) [inline backend/package.json]: Nope, damnit >_<

File: frontend/package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "react-responsive-carousel": "^3.2.23",
     "react-router-dom": "^6.3.0",
     "styled-components": "^5.3.5",
-    "styled-reset": "^4.3.4"
+    "styled-reset": "^4.3.4",
+    "template-fullstack": "file:.."
```
Comment (LoicBrassart) [inline frontend/package.json]: Nope

File: frontend/src/components/Header/style.jsx [hunk 3]
```diff
@@ -22,10 +22,12 @@ export default styled.header`
     position: fixed;
     width: 100vw;
     min-height: 75px;
+    transition: 0.5s;
+    z-index: 9999;
```
Comment (LoicBrassart) [inline frontend/src/components/Header/style.jsx]: You shouldn't use z-index like this

File: frontend/src/components/Header/style.jsx [hunk 4]
```diff
@@ -86,8 +88,12 @@ export default styled.header`
     position: relative;
     border-radius: 3px;
     background-color: #fff;
+    z-index: 999;
```
Comment (LoicBrassart) [inline frontend/src/components/Header/style.jsx:92]: >_>

File: frontend/src/components/Header/style.jsx [hunk 5]
```diff
@@ -118,17 +124,29 @@ export default styled.header`
...
+      background-color: black;
+    }
+    .navBarLogo {
+      padding-right: 1.3rem;
+    }
     .showNav .navBarLinks {
       left: 0;
       width: 100vw;
       visibility: visible;
+      z-index: 999;
```
Comment (LoicBrassart) [inline frontend/src/components/Header/style.jsx:150]: >_>

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Remove the "template-fullstack" dependency and it'll be ok

Advice: 
- Be careful with z-index, it **will** add problems later

**Comment by NtGZeko** [comment]
modif

**Review by ghost — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
Nice job on removing at least one `z-index` ! :muscle:

**Pull request merged**